core/src/gl/texture.cpp

```diff
@@ -8,6 +8,14 @@
 #include "tangram.h"
 #include "log.h"
 
+// Only JPEG, PNG and GIF
+#define STBI_NO_BMP
+#define STBI_NO_PSD
+#define STBI_NO_TGA
```

Comment:
TGA might be a good candidate to keep as it's likely faster to decode than PNG. For the other formats I don't see any real benefit for having them and it should be easy enough to export to our supported formats.